addons/default/visiosoft/base-theme/resources/views/addons/anomaly/users-module/login.twig

```diff
@@ -1,28 +1,31 @@
 {% extends "theme::layouts/default" %}
 
 {% block content %}
+    {% set phoneLogin = not setting_value('visiosoft.module.advs::register_email_field') or app.request.get('type') == 'phone' %}
     <div class="row justify-content-center">
         <div class="col-12 text-left m-0 row justify-content-center">
             <div class="col-12 col-lg-5 m-3 login-section">
                 {% set form = form('signIn')
                     .redirect(app.request.get('redirect'))
                     .get() %}
                 {{ form_open({
-                    'class':'aaa',
-                    'url':form.options.url
+                    'class': 'aaa',
```

Comment:
do not use meaningless classes. I will accept now but in the feature please be careful

Reply:
not my classes @ozcan-durak I noticed that but I didn't change it because I don't know where it has been used